app/upload/rest.py

```diff
@@ -74,3 +80,31 @@ def get_paginated_uploads(service_id, limit_days, page):
             service_id=service_id
         )
     }
+
+
+@upload_blueprint.route('/uploaded-letters/<letter_print_date>', methods=['GET'])
+def get_uploaded_letter_by_service_and_print_day(service_id, letter_print_date):
+    try:
+        letter_print_datetime = datetime.strptime(letter_print_date, '%Y-%m-%d')
+    except ValueError:
+        abort(404)
```

Comment:
This should be 400 response and raise a BadRequest

Reply:
Changed in cce153ee